src/transformers/models/instructblip/modeling_instructblip.py

```diff
@@ -1538,8 +1538,9 @@ def generate(
         inputs_embeds = torch.cat([language_model_inputs, inputs_embeds.to(language_model_inputs.device)], dim=1)
 
         # add image_embeds length to max_length, so that the final max_length in counted only on token embeds
+        # -1 is a temp hack until `generate` is refactored to work better with VLMs
```

Comment:
@gante's suggestion should also be applied here 

```suggestion
        # -1 is to account for the prepended BOS after `generate.`
```